Description: add boost libs to dbus to fix the armhf build
 The libdbus.a static library produced in the dbus/ directory needs some
 boost symbols, and for some reason this dependency isn't automatically
 detected when building for armhf. A similar fix I believe was done in
 server/Makefile.am in this commit:
 https://github.com/openSUSE/snapper/commit/33fbde9412d5eafb9f94aaff6b1be10bcda14951
Author: Andreas Hasenack <andreas@canonical.com>
Bug: https://github.com/openSUSE/snapper/issues/424
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/snapper/+bug/1789953
Last-Update: 2018-08-31
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
diff --git a/dbus/Makefile.am b/dbus/Makefile.am
index eb8769d..f9521a8 100644
--- a/dbus/Makefile.am
+++ b/dbus/Makefile.am
@@ -12,4 +12,4 @@ libdbus_la_SOURCES =					\
 	DBusMainLoop.cc		DBusMainLoop.h
 
 libdbus_la_LIBADD = $(DBUS_LIBS)
-
+libdbus_la_LDFLAGS = -lboost_system -lboost_thread
